### PR TITLE
Added Mohammad A. Islam from UT Arlington

### PR DESCRIPTION
**Pull request by mohammadatiqulislam**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -9953,6 +9953,7 @@ Mohamed Y. Eltabakh,Worcester Polytechnic Institute,http://www.wpi.edu/academics
 Mohamed Yehia Dahab,King Abdulaziz University,http://www.kau.edu.sa/Contacts.aspx?Site_ID=611&Lng=EN&Cont=18607,YExgqdMAAAAJ
 Mohamed Younis,Univ. of Maryland - Baltimore County,http://www.csee.umbc.edu/~younis,Q4MvxMIAAAAJ
 Mohamed Zahran,New York University,http://www.mzahran.com,Z2tBw84AAAAJ
+Mohammad A. Islam 001,University of Texas at Arlington,http://crystal.uta.edu/~mislam/,rxKWd1cAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Should be "0001" and needs an alias entry as well.
https://dblp.uni-trier.de/pers/hd/i/Islam_0001:Mohammad_A=

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 1 above)
Comment (mohammadatiqulislam) [reply csrankings.csv]: Updated accordingly in csrankings.csv and dblp-aliases.csv

File: dblp-aliases.csv [hunk 2]
```diff
@@ -2180,3 +2180,4 @@ Muhammad Zubair Shafiq,Zubair Shafiq
 Oscar Cánovas Reverte,Óscar Cánovas Reverte
 Óscar Cánovas,Óscar Cánovas Reverte
 Oznur Tastan,Öznur Tastan
+Mohammad A. Islam 0001
```
Comment (emeryberger) [inline dblp-aliases.csv]: Aliases come in pairs: alias, name

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: dblp-aliases.csv (hunk 2 above)
Comment (mohammadatiqulislam) [reply dblp-aliases.csv]: Alias and name updated in the right format.
I apologize that you had to ask for correction many times.

**Comment by emeryberger** [comment]
The updated version of `dblp-aliases.csv` contains a conflict. I am closing this PR and adding you manually.

**Pull request closed without merging**